apps/cowswap-frontend/src/modules/ordersTable/containers/OrdersTableWidget/index.tsx

```diff
@@ -50,6 +58,28 @@ const ContentWrapper = styled.div`
   width: 100%;
 `
 
+function useGetOrdersToCheckPendingPermit(
```

Comment:
Nitpick: to not grow the file size, would be great to move the hook into separate file